.travis.yml

```diff
@@ -44,7 +44,9 @@ before_cache:
 script:
   # Force git to update the shallow clone and include tags so git-describe works
   - git fetch --unshallow --tags
-  - sh autogen.sh && ./configure --with-flatbuffers=$HOME/flatbuffers && make -k && make DISTCHECK_CONFIGURE_FLAGS="--with-flatbuffers=$HOME/flatbuffers" distcheck
+  - sh autogen.sh
+  - ./configure --with-flatbuffers=$HOME/flatbuffers || cat config.log
```

Comment:
The flatcc package got merged into Spack yesterday, so this should be available in their latest develop branch.